### ISSUE TITLE
Can not init podman endpoint on Ubunutu

### ISSUE THREAD

**Issue by mancubus77** [skupperproject/skupper#1448]
**Describe the bug**

Podman can not find socket to initiate ingress on Ubuntu 22.04

**How To Reproduce**

Run tutorial example

```
$ podman run --name backend-target --network skupper --detach --rm -p 8080:8080 quay.io/skupper/hello-world-backend
366ceb8def672d36efcc2bd1ab21c26efac5d7d050ca1ec5145a86e667e43f4c
```

Init fails with message that socket is not available

```
skupper init --ingress none
Podman endpoint is not available: unix:///run/user/1001/podman/podman.sock

Recommendation:

	Make sure you have an active podman endpoint available.
	On most systems you can execute:

		systemctl --user enable --now podman.socket

	Alternatively you could also create your own service that runs:

		podman system service --time=0 <URI>

	You can get concrete examples through:

		podman help system service
```

Socket is available 

```
$ echo $XDG_RUNTIME_DIR/podman/podman.sock
/run/user/1001/podman/podman.sock

$ ls -la /run/user/1001/podman/podman.sock
srw------- 1 user user 0 апр 21 02:18 /run/user/1001/podman/podman.sock

$ curl --unix-socket $XDG_RUNTIME_DIR/podman/podman.sock http://localhost/version
{"Platform":{"Name":"linux/amd64/ubuntu-22.04"},"Components":[{"Name":"Podman Engine","Version":"3.4.4","Details":{"APIVersion":"3.4.4","Arch":"amd64","BuildTime":"1970-01-01T03:00:00+03:00","Experimental":"false","GitCommit":"","GoVersion":"go1.18.1","KernelVersion":"6.5.0-26-generic","MinAPIVersion":"3.1.0","Os":"linux"}},{"Name":"Conmon","Version":"conmon version 2.0.25, commit: unknown","Details":{"Package":"conmon: /usr/bin/conmon"}},{"Name":"OCI Runtime (crun)","Version":"crun version 0.17\ncommit: 0e9229ae34caaebcb86f1fde18de3acaf18c6d9a\nspec: 1.0.0\n+SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL","Details":{"Package":"crun: /usr/bin/crun"}}],"Version":"3.4.4","ApiVersion":"1.40","MinAPIVersion":"1.24","GitCommit":"","GoVersion":"go1.18.1","Os":"linux","Arch":"amd64","KernelVersion":"6.5.0-26-generic","BuildTime":"1970-01-01T03:00:00+03:00"}
```

 strace reports `EAGAIN (Resource temporarily unavailable)`

```
newfstatat(AT_FDCWD, "/run/user/1001/podman/podman.sock", {st_mode=S_IFSOCK|0600, st_size=0, ...}, 0) = 0
futex(0xc000110148, FUTEX_WAKE_PRIVATE, 1) = 1
socket(AF_UNIX, SOCK_STREAM|SOCK_CLOEXEC|SOCK_NONBLOCK, 0) = 6
connect(6, {sa_family=AF_UNIX, sun_path="/run/user/1001/podman/podman.sock"}, 36) = 0
epoll_ctl(3, EPOLL_CTL_ADD, 6, {events=EPOLLIN|EPOLLOUT|EPOLLRDHUP|EPOLLET, data={u32=2214414360, u64=132291502098456}}) = 0
getsockname(6, {sa_family=AF_UNIX}, [112 => 2]) = 0
getpeername(6, {sa_family=AF_UNIX, sun_path="///run/user/1001/podman/podman.sock"}, [112 => 38]) = 0
futex(0xc000110148, FUTEX_WAKE_PRIVATE, 1) = 1
read(6, 0xc0001ea000, 4096)             = -1 EAGAIN (Resource temporarily unavailable)
futex(0x25b2728, FUTEX_WAIT_PRIVATE, 0, NULL) = 0
epoll_pwait(3, [], 128, 0, NULL, 0)     = 0
futex(0x25b2728, FUTEX_WAIT_PRIVATE, 0, NULL) = 0
epoll_pwait(3, [], 128, 0, NULL, 0)     = 0
epoll_pwait(3, Podman endpoint is not available: unix:///run/user/1001/podman/podman.sock
```

**Environment details**
Skupper 

```
$ skupper version
client version                 1.6.0
```
podman
```
$ podman --version
podman version 3.4.4
```

OS
```
$ uname -r
6.5.0-26-generic
$ cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.4 LTS"
```

**Comment by fgiorgetti** [comment]
@mancubus77 thank you for reporting this issue.
The recommendation that is being displayed in your case is incorrect.
Skupper is designed to work with Podman V4+ only, and the recommendation
should have mentioned it, but instead it is incorrectly reporting that the endpoint
is not available. We need to fix this.

Podman v4+ is available with Ubuntu 23.10.

**Comment by mancubus77** [comment]
Thank you, closing the issue.
